### DISCUSSION TITLE
Making MahApps play nicely with VisualStudio designer in Uno Platform project

### DISCUSSION THREAD

**Discussion by progmars** [MahApps/MahApps.Metro#4340] [Q&A]
I'm not sure if the problem comes from MahApps or VisualStudio or Uno, but when I work with XAML designer, it complains about things that actually work just fine when I run the app:

![image](https://user-images.githubusercontent.com/4199737/186126620-28f7eaf0-0aac-437e-b719-8a64b3c4b463.png)

Is there any way to make Visual Studio see that it's complaining without any reason and that LinearGradientBrush is a valid value?

**Comment by MrCoolAndroid** [comment]
Where you able to use MahApps.Metro with Uno?

**Reply by timunie** [reply]
MahApps.Metro is WPF only afaik
